hw/system/occamy/WALKTHROUGH.md

```diff
@@ -0,0 +1,406 @@
+# Walkthrough
```

Comment:
This walkthrough is exellent :) but can we please move this to the proper documentation for the snitch repository (https://github.com/pulp-platform/snitch/tree/master/docs/ug) such that the content can be found in the official [snitch user guide](https://pulp-platform.github.io/snitch/ug/getting_started/) which gets compiled out of it?